### PR TITLE
.github: update pull request template

### PR DESCRIPTION
**Pull request by zz-jason**
### What problem does this PR solve? 

Not a problem. This PR aims to improve the usability of the pull request template.

### What is changed and how it works?

make the content more concise.

### Check List 

Tests 

 - No code

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
- - Need to cherry-pick to the release branch
- - Need to update the documentation
- - Need to update the `tidb-ansible` repository
+Please choose at least one of them to write the release note:
 
-Release note
+- **Release Note(in English)**:
 
- - Write release note for bug-fix or new feature.
+- **Release Note(in Chinese)**:
```
Comment (coocood) [inline .github/pull_request_template.md]: I think it's not appropriate.

Comment (siddontang) [reply .github/pull_request_template.md]: +1

Comment (wjhuang2016) [reply .github/pull_request_template.md]: I think we need more comment for `Release Note`, this may confuse contributor.

**Comment by coocood** [comment]
LGTM

**Comment by zz-jason** [comment]
the UT failure is recorded to https://github.com/pingcap/tidb/issues/15091

**Pull request merged**